self-core-impl/src/main/java/com/selfxdsd/core/managers/StoredProjectManager.java

```diff
@@ -38,6 +38,8 @@
  * @since 0.0.1
  * @todo #212:30min Test equals and hashcode methods for each Stored*
  *  classes (StoredProjectManager, StoredProject etc.).
+ *
+ * @todo #290:15min Create unit test for PmUser class.
```

Comment:
@ANadrowski This todo needs to have more words (10-15 words), otherwise 0pdd will complain. How about something like this: 

```
@todo #290:15min Unit tests have been added for class Authenticated. We should also
 add unit tests for class PmUser.
```

Reply:
ok :)